src/FEValues/face_values.jl

```diff
@@ -186,6 +100,52 @@ Return the normal at the quadrature point `qp` for the active face of the
 """
 getnormal(fv::FaceValues, qp::Int) = fv.normals[qp]
 
+nfaces(fv::FaceValues) = length(fv.geo_values)
+
+function checkface(fv::FaceValues, face::Int)
```

Comment:
Maybe the name is a bit generic. Can we be more specific (if we keep the function)